src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
```

Comment:
The margin is applied to all stars, including the first one. While you reset it in the `&__star:first-child` rule, it would be more efficient to not apply it in the first place to the first star. You could use the `:not(:first-child)` selector to achieve this.